From 64aca220361f52f316fe2cc5bbd92a932fc5d32e Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Wed, 15 Apr 2020 17:08:21 +0800
Subject: [PATCH] ARM: dts: rockchip: Add pstore pmsg config for rv1126

And set record-size to 128KB, console-size/pmsg-size to 256KB.

Change-Id: I690e6ed5050b8079a251eb05da94209b117144be
Signed-off-by: Tao Huang <huangtao@rock-chips.com>
---
 arch/arm/boot/dts/rv1126.dtsi | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/arm/boot/dts/rv1126.dtsi b/arch/arm/boot/dts/rv1126.dtsi
index 6a0292e9b8ca..7e3f5e138721 100644
--- a/arch/arm/boot/dts/rv1126.dtsi
+++ b/arch/arm/boot/dts/rv1126.dtsi
@@ -200,11 +200,11 @@
 
 		ramoops: ramoops@8000000 {
 			compatible = "ramoops";
-			reg = <0x8000000 0xf0000>;
-			record-size = <0x30000>;
-			console-size = <0xc0000>;
+			reg = <0x8000000 0x100000>;
+			record-size = <0x20000>;
+			console-size = <0x40000>;
 			ftrace-size = <0x00000>;
-			pmsg-size = <0x00000>;
+			pmsg-size = <0x40000>;
 			status = "disabled";
 		};
 	};
-- 
2.35.3

